components/compliance-service/reporting/relaxting/stats.go

```diff
@@ -40,15 +41,18 @@ func (backend ES2Backend) GetStatsSummary(filters map[string][]string) (*stats.R
 	}
 
 	LogQueryPartMin(queryInfo.esIndex, source, fmt.Sprintf("%s query", myName))
-
+	b, _ := json.Marshal(source)
+	fmt.Println(string(b))
```

Comment:
please use logger to print anything on logs

the same thing applies to all the other occurrences in this file